snips_nlu_metrics/metrics.py

```diff
@@ -10,19 +13,22 @@
 
 from snips_nlu_metrics.utils.constants import (
     AVERAGE_METRICS, CONFUSION_MATRIX, INTENTS, INTENT_UTTERANCES, METRICS,
-    PARSING_ERRORS, UTTERANCES)
+    UTTERANCES, PARSINGS)
 from snips_nlu_metrics.utils.exception import NotEnoughDataError
 from snips_nlu_metrics.utils.metrics_utils import (
     aggregate_matrices, aggregate_metrics, compute_average_metrics,
     compute_engine_metrics, compute_precision_recall_f1, compute_split_metrics,
     create_shuffle_stratified_splits)
 
+LOGGER = logging.getLogger(__name__)
+
 
 def compute_cross_val_metrics(
         dataset, engine_class, nb_folds=5, train_size_ratio=1.0,
         drop_entities=False, include_slot_metrics=True,
         slot_matching_lambda=None, progression_handler=None, num_workers=1,
-        seed=None, out_of_domain_utterances=None):
+        seed=None, out_of_domain_utterances=None,
+        persist_exact_parsings=False):
```

Comment:
I would rather use `include_exact_parsings` as nothing is persisted at this stage.